core/server/worker/src/main/java/alluxio/worker/grpc/DoraWorkerClientServiceHandler.java

```diff
@@ -109,14 +114,29 @@ public StreamObserver<ReadRequest> readBlock(StreamObserver<ReadResponse> respon
   public void getStatus(GetStatusPRequest request,
                         StreamObserver<GetStatusPResponse> responseObserver) {
     try {
-      String ufsFilePath = request.getPath();
-      UfsFileStatus status = mUfsFileStatusCache.get(request.getPath());
+      String alluxioFilePath = request.getPath();
+
+      String ufsFullPath = PathUtils.concatPath(mRootUFS,
+          CommonUtils.stripPrefixIfPresent(alluxioFilePath, mRootUFS));
+      String fn;
+      try {
+        String[] paths = PathUtils.getPathComponents(alluxioFilePath);
+        if (paths.length > 0) {
+          fn = paths[paths.length - 1];
+        } else {
+          fn = alluxioFilePath;
+        }
+      } catch (InvalidPathException e) {
+        fn = alluxioFilePath;
+      }
```

Comment:
We have `new AlluxioURI(alluxioFilePath).getName()` for this.

Reply:
Fixed